CMakeLists.txt

```diff
@@ -1,21 +1,13 @@
 cmake_minimum_required(VERSION 3.0)
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib/linux)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib/linux)
-set(CMAKE_BUILD_TYPE Debug)
+if (NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Debug)
+endif()
```

Comment:
it's best not to force your build type to Debug. This way, you can generate another build folder for `Release` without having to change your CMakeLists.txt